.github/workflows/main.yaml

```diff
@@ -158,3 +131,26 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: coverage.xml
           fail_ci_if_error: true
+
+  test_build_package:
+    name: Test Build Package
+    needs:
+      - test
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.13
+
+      - name: Install pipx and Poetry with dynamic versioning plugin
+        run: |
+          pipx ensurepath
+          pipx install poetry==1.*
+          pipx inject poetry poetry-dynamic-versioning
```

Comment:
for poetry 1.x the plugin should be installed with:
```
poetry self add "poetry-dynamic-versioning[plugin]"
```
Currently the version number is still 0.0.0 in https://github.com/linkml/linkml/actions/runs/14453483851/job/40532713312?pr=2617

Reply:
Not sure if the "[plugin]" changes anything...